src/ScottPlot4/ScottPlot/Plottable/ScatterPlot.cs

```diff
@@ -422,6 +423,30 @@ public void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false)
                     DrawLinesWithGaps(points, gfx, penLine);
                 }
 
+                if (DataPointLabels != null)
+                {
+                    for (var i = 0; i < DataPointLabels.Length; i++)
+                    {
+                        var label = DataPointLabels[i];
+                        if (!string.IsNullOrEmpty(label))
+                        {
+                            Drawing.Font textFont = new();
+                            var stringSize = GDI.MeasureString(gfx, label, textFont);
+                            gfx.TranslateTransform(points[i].X, points[i].Y);
+                            gfx.RotateTransform(textFont.Rotation);
```

Comment:
I believe `textFont.Rotation` is always zero, so this line is redundant (unless we decide to add customization later)

Reply:
If we expose `DataPointLabelFont` as described earlier, rotation could be customized, so let's leave this like it is 👍